src/main/java/uiPages/ProductPage.java

```diff
@@ -0,0 +1,70 @@
+package uiPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import helpers.WebElementHelper;
+
+import static helpers.WebElementHelper.areVisible;
+import static helpers.WebElementHelper.setFieldValue;
+
+public class ProductPage extends BasePage {
+
+    private static final String URL_PATH = "/product";
+
+    @FindBy(className = "product-essential__description")
+    private WebElement productDescriptionField;
+
+    @FindBy(className = "product-essential__specification")
+    private WebElement productSpecificationField;
+
+    @FindBy(xpath = "//button[@class='single_add_to_cart_button button alt disabled wc-variation-selection-needed']")
```

Comment:
done
I wanted to identify each button separately, if I only identify the button by its class how can I identify when it's disabled?

Reply:
We map the page's html elements in our page class once. It's not indicated to have 2 or more WebElement variables for the same html element. In case of an element which has different states (e.g. disabled / enable), you can use ExpectedConditions class (https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/support/ui/ExpectedConditions.html). There are plenty of methods which allow to identify when an element is clickable or not, if the web element has a certain attribute or not, etc.
In add to cart button case you can use the ExpectedConditions.elementToBeClickable or ExpectedConditions.not(ExpectedConditions.attributeContains(element, attribute, value) (where attribute is "class" and value "disabled") to identify when the button is enabled. 
Since this method (of waiting for a element to become enabled) could be used in the future in other tests context, it's indicated to create a general method in the WebUIPage (see waitForElementToDisappear) which can be used by any other element in the app.